src/devices/Board/samples/Board.Sample.csproj

```diff
@@ -4,11 +4,13 @@
     <OutputType>Exe</OutputType>
     <TargetFramework>$(DefaultSampleTfms)</TargetFramework>
     <LangVersion>preview</LangVersion>
+    <EnableDefaultItems>false</EnableDefaultItems>
```

Comment:
any reason for explicit?

Reply:
Yes, because there are now 2 sample projects. But I can definitely merge both. Thoughts?

Reply:
merged both samples finally, it's easier